### PR TITLE
Feature/add material manager client edgebands and adjust boards

### PR DESCRIPTION
**Pull request by HOMAG-Robin-Katzenberger**

### CODE REVIEW

File: Applications/MaterialManager/Client/MaterialManagerClientMaterialBoards.cs [hunk 1]
```diff
@@ -11,33 +11,50 @@
 
 namespace HomagConnect.MaterialManager.Client;
 
+/// <summary>
+/// materialMaterial Client for boards
```
Comment (SchaberMedien) [inline Applications/MaterialManager/Client/MaterialManagerClientMaterialBoards.cs:15]: materialManager

File: Applications/MaterialManager/Contracts/Material/Base/ImageInformation.cs [hunk 2]
```diff
@@ -4,22 +4,49 @@
...
 
+        /// <summary>
+        /// Gets or sets the file name.
+        /// </summary>
         public string? FileName { get; set; }
 
+        /// <summary>
+        /// Gets or sets the content type.
+        /// </summary>
         public string? ContentType { get; set; }
```
Comment (SchaberMedien) [inline Applications/MaterialManager/Contracts/Material/Base/ImageInformation.cs:35]: Shouldn't the content type be an enum?

File: Applications/MaterialManager/Contracts/Material/Edgebands/Interfaces/IMaterialManagerClientMaterialEdgebands.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace HomagConnect.MaterialManager.Contracts.Material.Edgebands.Interfaces
+{
+    /// <summary>
+    /// Interface for MaterialManager Edgebands Client.
+    /// </summary>
+    public interface IMaterialManagerClientMaterialEdgebands
```
Comment (SchaberMedien) [inline Applications/MaterialManager/Contracts/Material/Edgebands/Interfaces/IMaterialManagerClientMaterialEdgebands.cs:9]: sure about the naming? 2 time Material.
Maybe: IMaterialManagerClientEdgebands

**Review by SchaberMedien — DISMISSED** [review]

File: Applications/MaterialManager/Contracts/Material/Edgebands/Interfaces/IMaterialManagerClientMaterialEdgebands.cs (hunk 3 above)
Comment (HOMAG-Robin-Katzenberger) [reply Applications/MaterialManager/Contracts/Material/Edgebands/Interfaces/IMaterialManagerClientMaterialEdgebands.cs:9]: @HOMAG-Boris-Wehrle suggested this from the start
I think the background is that there will be processing and the client for that will be called
IMaterialManagerClientProcessingOffcutParameters

Comment (SchaberMedien) [reply Applications/MaterialManager/Contracts/Material/Edgebands/Interfaces/IMaterialManagerClientMaterialEdgebands.cs:9]: It is okay, just wanted to be sure that it is intended.

File: Applications/MaterialManager/Contracts/Material/Base/ImageInformation.cs (hunk 2 above)
Comment (HOMAG-Robin-Katzenberger) [reply Applications/MaterialManager/Contracts/Material/Base/ImageInformation.cs:35]: we don´t have an enum for that. Are the values all known?

Comment (SchaberMedien) [reply Applications/MaterialManager/Contracts/Material/Base/ImageInformation.cs:35]: Or maybe I confused it wiht Type?

We have at least these:
image

and this 
image

I think now is the chance if we want to have them as enums for the API. I think this makes checks and switches easier and more reliable.

File: Applications/MaterialManager/Contracts/Material/Base/ImageType.cs [hunk 4]
```diff
@@ -0,0 +1,38 @@
+﻿namespace HomagConnect.MaterialManager.Contracts.Material.Base
+{
+    /// <summary>
+    /// Image types
+    /// </summary>
+    public enum ImageType
```
Comment (SchaberMedien) [inline Applications/MaterialManager/Contracts/Material/Base/ImageType.cs:6]: @HOMAG-Boris-Wehrle should we extend and maybe align with types from pM and iD 
image

Comment (HOMAG-Robin-Katzenberger) [reply Applications/MaterialManager/Contracts/Material/Base/ImageType.cs:6]: If necessary please resolve it in a different PR where the models that should be globally available will be moved to the Base Contracts and extended by the necessary values

**Review by aldfhg — APPROVED** [review]

**Review by SchaberMedien — APPROVED** [review]

**Pull request merged**